sycl/test-e2e/CMakeLists.txt

```diff
@@ -41,6 +41,23 @@ if(NOT SYCL_TEST_E2E_TARGETS)
   set(SYCL_TEST_E2E_TARGETS "all")
 endif()
 
+if(MSVC)
```

Comment:
Can we additionally guard it with `NOT SYCL_TEST_E2E_STANDALONE`? I'd prefer to fail instead of doing some magic when passing wrong options exclusively to the E2E CMake invocation.

Reply:
I think it would be useful in the standalone case as well, but I've updated it in order to ensure the internal CI doesn't change: https://github.com/intel/llvm/pull/11053/commits/7017cccf246e9b0170eee01e8055c66d05d689c2